microprofile.ls/org.eclipse.lsp4mp.ls/src/test/java/org/eclipse/lsp4mp/services/properties/PropertiesFileCompletionTest.java

```diff
@@ -355,7 +355,13 @@ public void completionDefaultValueContainsDollarSignAndBraces() throws BadLocati
 	@Test
 	public void completionBetweenPropertyNameAndEquals() throws BadLocationException {
 		String value = "quarkus.http.cors|=";
-		testCompletionFor(value, true, 0);
+		testCompletionFor(value, true, 1489);

```

Comment:
Please don't hard code the number of expected completion items, because if we add another properties, your test will fail. In our test we use `0` to test that we have none completion. But in your case you expect that you have some completion items. You don't need to check the number of completion items, but just check you have some completion items.

I suggest that you change with (by adjusting the range r(0, 0, 20)):

```
testCompletionFor(value, false,
				c("quarkus.log.category.{*}.level", "quarkus.log.category.{*}.level=inherit", r(0, 0, 20)));
```

I have a question, when you have that 

`greeting.messag|=abcd`

when you apply completion, do you loose the content after `|` ? I mean do you have that:

`greeting.message`

or 

`greeting.message=abcd` ?

You should have the last.

Reply:
It will replace the message after according to the default value. 

![completionBeforeAssign](https://user-images.githubusercontent.com/73968480/176700417-6510af3d-e508-45c6-8558-4594e4280de4.gif)

Reply:
Indeed it was my fear, I think we should not update the value (random_name) when completion is applied.

Reply:
Ok let's do that in a separate issue.